script/bump-version.py

```diff
@@ -194,19 +204,12 @@ def update_info_plist(version):
 
 
 def update_package_json(version, suffix):
-  package_json = 'package.json'
```

Comment:
@harryi3t Did we find anything where just editing the version on package.json was insufficient?

Reply:
@kamalaknn This logic is just updating the package.json but not the lock file. Even the version in the lock file on `3-1-x` branch from which we forked is outdated.

Also, didn't want to latch onto the same logic of reading file content and the find the pattern and replacing that.
AFAIK, the safest and recommended way to update the version in both package.json and lock file is to use the npm inbuilt command itself.